custom_components/senseme/aiosenseme/device.py

```diff
@@ -0,0 +1,1266 @@
+"""SensemeFan Class.
```

Comment:
This file will be removed once https://github.com/mikelawrence/aiosenseme/pull/11 is merged